tests/Tests/Isolated/Common/Twig/fixtures/render/amc-full-report-non-root-webroot.html

```
@@ -0,0 +1,69 @@
+<html>
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)